src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -11809,6 +11809,7 @@ public enum ImageFileMachine
         ARM = 452,
         IA64 = 512,
         AMD64 = 34404,
+        LOONGARCH64 = 25188,
```

Comment:
The changes in this file should be reverted.

Reply:
OK,Thanks for your review~